app/javascript/gobierto_data/webapp/components/commons/RecentQueries.vue

```diff
@@ -36,21 +36,23 @@ export default {
     }
   },
   created() {
-    document.addEventListener("keyup", this.onKeyUpNextRecentItem)
+    document.addEventListener("keydown", this.onKeyUpNextRecentItem)
   },
   beforeDestroy() {
-    document.removeEventListener("keyup", this.onKeyUpNextRecentItem)
+    document.removeEventListener("keydown", this.onKeyUpNextRecentItem)
```

Comment:
Since you've renamed the event to `keydown`, update the function names accordingly